pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/BookkeeperSchemaStorage.java

```diff
@@ -213,7 +213,7 @@ private CompletableFuture<StoredSchema> getSchema(String schemaId) {
                 return readSchemaEntry(schemaLocator.getInfo().getPosition())
                         .thenApply(entry -> new StoredSchema(entry.getSchemaData().toByteArray(),
                                 new LongSchemaVersion(schemaLocator.getInfo().getVersion())));
-            }).handleAsync((res, ex) -> {
+            }).handle((res, ex) -> {
```

Comment:
@Technoboy- There were 2 instances of `handleAsync()` in the original commit, should we change both?

Reply:
Ah, `getOrCreateSchemaLocator` not existed in the master, so we only have one instance now.